modules/contacts/client/api/contacts.api.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+/**
+ * Delete a contact.
+ */
+export async function remove(contactId) {
+  await axios.delete(`/api/contact/${contactId}`);
+}
+
+export async function contactsCommonListService(id) {
```

Comment:
Perhaps a name like `getContactsCommon` or similar would suit this function? The name of the old angular service doesn't necessarily express the purpose of the function. A nitpick.

Reply:
Thanks! :) I hope you don't mind these little whims. Otherwise please do complain! :slightly_smiling_face: